vendor.conf

```diff
@@ -8,7 +8,7 @@ github.com/matttproud/golang_protobuf_extensions v1.0.0
 github.com/grpc-ecosystem/go-grpc-prometheus 6b7015e65d366bf3f19b2b2a000a831940f0f7e0
 
 # etcd/raft
-github.com/coreos/etcd 824277cb3a577a0e8c829ca9ec557b973fe06d20
+github.com/coreos/etcd ea5389a79f40206170582c1ea076191b8622cb8e https://github.com/aaronlehmann/etcd
```

Comment:
nit: does it make sense to link to the specific etcd PR here, so that it's clear why this fork was needed?